ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -705,294 +705,295 @@
+        'Refulgent Fate': 'Lien de Lumière',
+        'Thin Ice': 'Verglas',
+        'Wyrmclaw': 'Griffes du Dragon divin',
+        'Wyrmfang': 'Crocs du Dragon divin',
       },
     },
     {
       'locale': 'ja',
       'missingTranslations': true,
       'replaceSync': {
-        'luminous Aether': 'ライト・エーテル',
-        'holy light': '聖なる光',
-        'great wyrm': '聖竜',
-        'frozen mirror': '氷面鏡',
-        'electric Aether': 'ライトニング・エーテル',
-        'earthen Aether': 'アース・エーテル',
         'aqueous Aether': 'ウォーター・エーテル',
-        'Shiva': 'シヴァ',
+        'earthen Aether': 'アース・エーテル',
+        'electric Aether': 'ライトニング・エーテル',
+        'frozen mirror': '氷面鏡',
+        'great wyrm': '聖竜',
+        'holy light': '聖なる光',
+        'luminous Aether': 'ライト・エーテル',
         'Mothercrystal': 'マザークリスタル',
+        'Shiva': 'シヴァ',
       },
       'replaceText': {
+        'attack': '攻撃',
+        'Absolute Zero': '絶対零度',
+        'Akh Morn': 'アク・モーン',
+        'Akh Rhai': 'アク・ラーイ',
+        'Axe/Scythe Kick': 'アクスキック/サイスキック',
+        'Banish(?! )': 'バニシュ',
+        'Banish Divided': 'ディバイデッド・バニシュ',
+        'Banish III': 'バニシュガ',
+        'Banish III Divided': 'ディバイデッド・バニシュガ',
+        'Biting/Driving Frost': 'フロストスラッシュ/フロストスラスト',
+        'Bright Hunger': '浸食光',
+        'Bright Pulse': '閃光',
+        'Diamond Frost': 'ダイヤモンドダスト',
+        'Double Slap': 'ダブルスラップ',
+        'Drachen Armor': 'ドラゴンアーマー',
+        'Draconic Strike': 'ドラコニックストライク',
+        'Driving/Biting Frost': 'フロストスラスト/フロストスラッシュ',
+        'Embittered/Spiteful Dance': 'Embittered/冷厳の舞踏技',
+        'Frigid Eruption': 'アイスエラプション',
+        'Frigid Needle': 'アイスニードル',
+        'Frigid Stone': 'アイスストーン',
+        'Frigid Water': 'アイスフロスト',
+        'Frost Armor(?! )': 'フロストアーマー',
+        'Hallowed Wings': 'ホーリーウィング',
+        'Heart Asunder': 'ハートアサンダー',
+        'Heavenly Strike': 'ヘヴンリーストライク',
+        'Holy': 'ホーリー',
+        'Icelit Dragonsong': '氷と光の竜詩',
+        'Icicle Impact': 'アイシクルインパクト',
+        'Light Rampant': '光の暴走',
+        'Longing of the Lost': '聖竜気',
+        'Mirror, Mirror': '鏡の国',
+        'Morn Afah': 'モーン・アファー',
+        'Reflected Biting Frost': 'ミラーリング・フロストスラッシュ',
+        'Reflected Drachen Armor': 'ミラーリング・ドラゴンアーマー',
+        'Reflected Frost Armor': 'ミラーリング・フロストアーマー',
+        'Reflected Hallowed Wings': 'ミラーリング・ホーリーウィング',
+        'Reflected Scythe Kick': 'ミラーリング・サイスキック',
+        'Reflected Shining Armor': 'ミラーリング・ブライトアーマー',
+        'Rush': 'ラッシュ',
+        'Scythe/Axe Kick': 'サイスキック/アクスキック',
+        'Shattered World': 'シャッタード・ワールド',
+        'Shining Armor': 'ブライトアーマー',
+        'Shock Spikes': 'ショックスパイク',
+        'Skyfall': '世界消滅',
+        'Stoneskin': 'ストンスキン',
+        'Spiteful/Embittered Dance': '冷厳の舞踏技/Embittered Dance',
         'the Path of Light': '光の波動',
         'the House of Light': '光の津波',
-        'attack': '攻撃',
+        'Twin Silence/Stillness': 'Twin Silence/静寂の双剣技',
+        'Twin Stillness/Silence': '静寂の双剣技/Silence',
```

Comment:
Need help for fix plz. @maiko-tan-coding  @Jaehyuk-Lee

Reply:
```javascript
'Twin Silence/Stillness': '閑寂の双剣技／静寂の双剣技',
'Twin Stillness/Silence': '静寂の双剣技／閑寂の双剣技',
```